### PR TITLE
Added option to disable the catkin build

### PR DESCRIPTION
**Pull request by goldhoorn**

### CODE REVIEW

**Comment by goldhoorn** [comment]
ping

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,12 @@ cmake_minimum_required(VERSION 2.8.3)
 project(class_loader)
 
 find_package(Boost REQUIRED COMPONENTS thread system)
-find_package(catkin QUIET)
+
+set(CATKIN_DISABLED false CACHE BOOL "Disable the catkin build, usefull if catkin is present but a build outside or ros is done" )
```
Comment (esteve) [inline CMakeLists.txt]: Typo in "usefull"

**Comment by esteve** [comment]
+1, please fix the typo and squash the commits. Thanks!

**Comment by goldhoorn** [comment]
updated and force-pushed as one commit

**Comment by esteve** [comment]
Thanks!

**Pull request merged**